runtime/runtime/src/lib.rs

```diff
@@ -1115,13 +1112,27 @@ impl Runtime {
             state_root,
             trie_changes,
             validator_proposals,
-            new_receipts,
+            new_receipts: outgoing_receipts,
```

Comment:
Shouldn't we rename `ApplyResult.new_receipts` to `ApplyResult.outgoing_receipts` too?

Reply:
Sure. It looks like it doesn't go too far with this naming.